tests/attr/neuron/test_neuron_deeplift_basic.py

```diff
@@ -114,8 +118,22 @@ def test_relu_deepliftshap_with_custom_attr_func(self):
         expected = (torch.zeros(3, 3), torch.zeros(3, 3))
         self._relu_custom_attr_func_assert(neuron_dl, inputs, baselines, expected)
 
-    def _relu_custom_attr_func_assert(self, attr_method, inputs, baselines, expected):
-        def custom_attr_func(multipliers, inputs, baselines):
+    def _relu_custom_attr_func_assert(
+        self,
+        attr_method,
+        inputs: TensorOrTupleOfTensors,
+        baselines: Optional[
+            Union[Tensor, int, float, Tuple[Union[Tensor, int, float], ...]]
+        ],
+        expected,
+    ) -> None:
+        def custom_attr_func(
+            multipliers: TensorOrTupleOfTensors,
```

Comment:
multipliers, inputs, and the return type here should be Tuples of Tensors, while baselines should be a Tuple of either Tensor, int or float

Reply:
Sorry, I think this might not have been clear, these should just be Tuple[Tensor, ...] not TensorOrTupleOfTensors and baselines should be just Optional[Tuple[Union[Tensor, int, float], ...]]

Reply:
Thanks!